examples/demo-rollup/README.md

```diff
@@ -226,15 +226,19 @@ Let's go ahead and import the transaction into the wallet
 $ cargo run --bin sov-cli -- transactions import from-file bank --path ../test-data/requests/transfer.json
```

Comment:
The `chains_id` and `gas_tip` parameters are not provided to the CLI. Where are these values obtained?

Reply:
This is the stdout printed here

https://github.com/Sovereign-Labs/sovereign-sdk/blob/3ae9b7b92c8123864bf7471bfb4c086da93fe3af/module-system/sov-cli/src/workflows/transactions.rs#L136

I opted to add the full `UnsignedTransaction` because `(chain_id, gas_tip)` are part of the user input, so it makes sense to integrate these into the feedback

Reply:
My question is how these parameters are inserted as  there are no `--chain_id` & `--chain_id` parameters in the above commands.

Reply:
They have a default value of `0`, as of the clap implementation

https://github.com/Sovereign-Labs/sovereign-sdk/blob/3ae9b7b92c8123864bf7471bfb4c086da93fe3af/module-system/sov-modules-api/src/cli.rs#L27-L29

Reply:
Ahh, I think we should not have default values for this parameter as someone could by accident send a tx to a wrong chain.

Reply:
Done on https://github.com/Sovereign-Labs/sovereign-sdk/pull/1213/commits/32c6a8f33c234fcdc428f56036fef8d13e804680